pages/common/verilator.md

```diff
@@ -0,0 +1,31 @@
+# verilator
+
+> Converts Verilog and SystemVerilog hardware description language (HDL) designs into a C++ or SystemC model that after compiling can be executed.
+> More information: <https://veripool.org/ftp/verilator_doc.pdf>.
+
+Example Verilog file:
+
+```bash
+mkdir test_our
+cd test_our
+cat >our.v <<'EOF'
+module our;
+initial begin $display("Hello World"); $finish; end
+endmodule
+EOF
+```
```

Comment:
This is an invalid example for `tldr-pages`.

Reply:
Removed.